woocommerce/wc-admin
# Orders table

Part of #418.

**Changes**
* Add a _Orders last week_ table to the _Orders_ page in _Analytics_.
* CSV download now removes commas from values.
* All table values are now aligned to the right except the ones marked as identifiers. Until now, we were only aligning to the right numeric values, but that didn't seem to follow the designs.
* Set `th`'s `font-weight` to `normal` as it is in the designs.
* Other minor fixes with RTL and tables.

**Screenshot**
![image](https://user-images.githubusercontent.com/3616980/46668302-49fec380-cbcc-11e8-8d75-0351261df2a8.png)

**Steps to test**
* Go to _Analytics_ > _Orders_.
* Verify the table with the latest orders is there and works as expected.
* Try sorting rows, downloading its contents, clicking on the links, etc.

**Work for follow-up PRs**
This PR was already getting big so I will continue working on this in follow-up PRs for this features:
* Pagination (#519).
* `ViewMore` Component for long lists (#503).

**Future changes (can't be implemented yet)**
* We are currently displaying the customer ID instead of the customer type (new/returning). There is a `@TODO` comment in the code so we can implement it once it's in the API.
* The coupon will have to link to the coupon report (added a `@TODO` comment to the code as well).
* Totals in the footer of the table.
